libs/frontend/src/lib/components/ui/container/container.svelte

```diff
@@ -8,6 +8,6 @@
 	export { className as class };
 </script>
 
-<div class={cn("container mx-auto my-4 flex flex-col", className)}>
+<div class={cn("container mx-auto my-4 flex max-w-screen-lg flex-col", className)}>
```

Comment:
change the margin possibly to padding and increase it for larger screen sixes :+1: